examples/window-app/efr32/src/main.cpp

```diff
@@ -84,7 +85,20 @@ void appError(CHIP_ERROR err)
     while (1)
         ;
 }
+#if CHIP_ENABLE_OPENTHREAD
+// ================================================================================
+// Matter Networking Callbacks
+// ================================================================================
+void LockOpenThreadTask(void)
+{
+    chip::DeviceLayer::ThreadStackMgr().LockThreadStack();
+}
```

Comment:
A lot of the common code for all examples, especially regarding init, could be added to a file in `examples/platform/efr32` to reduce the amount of cut'n'paste. Most of your `main.cpp` have nearly identical boilerplate that could be factored out.

Reply:
Will clean it up